client/ayon_core/tools/publisher/control.py

```diff
@@ -1839,6 +1853,45 @@ def _collect_creator_items(self):
 
         return output
 
+    def _get_allowed_creator_identifiers(
+            self,
+            project_name,
+            host_name,
+            task_name,
+            task_type,
+            log=None
+    ):
+        """Provide configured creator identifier in this context
+
+        If no profile provided for current context, it shows all creators
+        """
+        allowed_creator_identifiers = []
+        if host_name == "traypublisher":
+            # no real context known
+            return allowed_creator_identifiers
+        proj_settings = get_project_settings(project_name)
+        filter_creator_profiles = (
+            proj_settings
+            ["core"]
+            ["tools"]
+            ["creator"]
+            ["filter_creator_profiles"]
```

Comment:
Should we be careful with someone running an older server addon but a newer client potentially? Or is that more of an issue for e.g. `houdini` etc. because they are included within ayon core?
```suggestion
            .get("filter_creator_profiles", [])
```

Reply:
> I had seen a lot of fallbacks being applied in other PRs, but maybe that was because the client facing code was ayon_core being sepately updated from e.g. ayon_houdini server addon - so I guess you can ignore this, yes.

Yep, the fallbacks are in hosts and modules, which have settings in `server_addons` folder.